docs/read-the-docs/source/workceptor.rst

```diff
@@ -0,0 +1,182 @@
+Workceptor
+==========
+
+Workceptor is a component of receptor that handles units of work.
+
+``work-command`` defines a type of work that can run on the node.
+
+foo.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: foo
+
+    - log-level:
+        level: Debug
+
+    - tcp-listener:
+        port: 2222
+
+    - control-service:
+        service: control
+        filename: /tmp/foo.sock
+
+    - work-command:
+        workType: echosleep
+        command: bash
+        params:  "-c \"for i in {1..5}; do echo $i; done\""
```

Comment:
The `echosleep` command in the example `echo`s, but does not actually `sleep`

Reply:
(`bar`'s `echosleep` does actually sleep)

Reply:
fixed, thanks